client/site-profiler/components/hosting-information/index.tsx

```diff
@@ -16,6 +18,21 @@ export default function HostingInformation( props: Props ) {
 	const aRecordIps = dns.filter( ( x ) => x.type === 'A' && x.ip );
 	const supportUrl = useHostingProviderURL( 'support', hostingProvider, urlData );
 
+	useEffect( () => {
+		hostingProvider &&
+			recordTracksEvent( 'calypso_site_profiler_hosting_information', {
+				domain_url: urlData?.url,
+				is_cdn: hostingProvider?.is_cdn,
+				provider_slug: hostingProvider?.slug,
+				provider_name: hostingProvider?.name,
+				platform: urlData?.platform,
+				platform_slug: urlData?.platform_data?.slug,
+				platform_is_wpcom: urlData?.platform_data?.is_wpcom,
+				platform_is_wpengine: urlData?.platform_data?.is_wpengine,
+				platform_is_pressable: urlData?.platform_data?.is_pressable,
```

Comment:
This event might be triggered before the `urlData` data comes back. Maybe we split it into two events here? Another call something like `calypso_site_profiler_platform_information` ?

Reply:
Good catch, I will keep the same event but improve the check by adding "hostingProvider && urlData".